### ISSUE TITLE
Bug: Output device reset to MacBook Pro Speakers after 10-20 minutes of idling

### ISSUE THREAD

**Issue by LAJW** [bitgapp/eqMac#632]
- [x] I have checked for a similar issue and sure it hasn't been reported before.

**Describe the bug**
Output device of eqmac resets to "MacBook Pro Speakers". When selecting the headphones again, they're set to ~80% volume every time. Volume slider is not responsive.

**Steps to Reproduce**
1. Connect and select bluetooth headphones as audio out
2. Let the machine idle for 10-20 minutes

**Expected behaviour**
Bluetooth headset should still be selected (it is connected to the machine). Volume should not change. Volume slider should be responsive.

**Setup information:**
 - Audio device used for playback: Sennheiser Momentum M2 IE Wireless
 - Audio transmission interface: Bluetooth
 - macOS Version: 12.1
 - eqMac Version 1.4.4
 - eqMac UI Version 3.2.4

**Additional information**
In the previous version the volume slider would get unresponsive. Now it starts unresponsive and gets better after a while, but every reset makes all volume silders unresponsive (except headphone controls on the device). These headphones don't have a sleep function. They are selected as the default output device with EQmac off. They also have a minimum/maximum sound (1%) which seems to trip up the app.

**Comment by nikolaydyankov** [comment]
Agree. I have speakers connected to my Mac Mini and every time it wakes up I have to reset the audio sources because the audio interface is not available in the menu, then select the audio interface.

Also the Spatial Audio is weird, have to select another preset and go back to the one previously selected.

**Comment by everykindofpeople** [comment]
M1 Mac Mini here. When using eqMac. Every time I log off, when logged back in, Audio output device has gone back to the Mac Mini speakers, and I need to reselect my AV amplifier. Every time.

**Comment by mikhail-fedosenko** [comment]
@superjeng1 Hi, it seems i have very same problem... eqMac successfully switches to external headphones when connected, but fails to autoswitch to USB Audio device from the list.
I've tried reordering as you've suggested:
1. headphones
2. USB audio device (Xiaomi)
3. Macbook pro speakers
4. Art M DisplayPort audio

I've recorded a video illustrating this problem: first, i've started with internal speakers, then manually switched to Xiaomi (usb audio device), then plugged in headphones => it switched to headphones automatically. Then i've unplugged the headphones => it switched to internal speakers, though they have lower priority than Xiaomi device (USB audio device).

Video: https://www.dropbox.com/scl/fi/bp3afdb85cqskjvlttvq9/eqMac-no-autoswitch-to-usb-device-2025-09-30-17.49.16.mov?rlkey=m2yn2xu8sc1jijk7bhaismsra&st=itmsmov6&dl=0

App version 1.8.11
<img width="370" height="62" alt="Image" src="https://github.com/user-attachments/assets/101da9b3-dd33-40fe-92a5-a866a778ed9c" />
